OrbitQt/orbitsamplingreport.cpp

```diff
@@ -44,20 +44,20 @@ void OrbitSamplingReport::Initialize(DataView* callstack_data_view,
 
   m_SamplingReport->SetUiRefreshFunc([&]() { this->Refresh(); });
 
-  for (SamplingReportDataView& report : report->GetThreadReports()) {
+  for (SamplingReportDataView& reportDataView : report->GetThreadReports()) {
```

Comment:
report_data_view?

Reply:
or maybe not since everything is camelcase here...

Reply:
Ah wow, yes, done.

Reply:
> or maybe not since everything is camelcase here...

Yeah, that's why I automatically used camelCase without thinking...

Reply:
Still, let's stick to the style guide, `callstack_data_view` is not camelCase after all!